experimental/dds/tree2/src/test/shared-tree/fuzz/anchorStability.fuzz.spec.ts

```diff
@@ -156,37 +152,39 @@ describe("Fuzz - anchor stability", () => {
 		};
 		const generatorFactory = () =>
 			takeAsync(opsPerRun, makeOpGenerator(editGeneratorOpWeights));
-		const generator = generatorFactory() as AsyncGenerator<Operation, AbortFuzzTestState>;
+		const generator = generatorFactory() as AsyncGenerator<Operation, AnchorFuzzTestState>;
 		const model: DDSFuzzModel<
 			SharedTreeTestFactory,
 			Operation,
 			DDSFuzzTestState<SharedTreeTestFactory>
 		> = {
 			workloadName: "anchors-undo-redo",
-			factory: new SharedTreeTestFactory(onCreate),
+			factory: new SharedTreeTestFactory(() => undefined),
 			generatorFactory: () => generator,
 			reducer: fuzzReducer,
 			validateConsistency: () => {},
 		};
 
 		const emitter = new TypedEventEmitter<DDSFuzzHarnessEvents>();
-		emitter.on("testStart", (initialState: AbortFuzzTestState) => {
+		emitter.on("testStart", (initialState: AnchorFuzzTestState) => {
 			initialState.anchors = [];
+			initialState.views = [];
 			for (const client of initialState.clients) {
-				const view = fuzzViewFromTree(client.channel) as RevertibleSharedTreeView;
+				const view = client.channel.schematizeView(config) as RevertibleSharedTreeView;
```

Comment:
the two clients will now race to set up the initial tree. This might make the losers set of anchors less interesting. Not really sure what the intent of this bit is though (maybe that's fine and the first clients' anchors are sufficient validation?)

Reply:
I'm also unclear on what this is supposed to be testing. I added docs for the parts I figured out, but like you I don't know if this change is a problem. The logic being tested here is that anchors and the forest stay in sync with eachother (within a client not between clients). Collab is only relevant if maybe we produce more complex editing cases by leveraging conflicts and rebasing, so I think this is fine.